common/src/lib.rs

```diff
@@ -3,5 +3,46 @@
 //! # SBTC Common Library
 //!
 //! This library provides common functionality for the sBTC project, including logging setup
+use std::sync::OnceLock;
 
+use bitcoin::XOnlyPublicKey;
+
+pub mod deposits;
 pub mod logging;
+
+/// The x-coordinate public key with no known discrete logarithm.
+///
+/// # Notes
+///
+/// This particular X-coordinate was discussed in the original taproot BIP
+/// on spending rules BIP-0341[1]. Specifically, the X-coordinate is formed
+/// by taking the hash of the standard uncompressed encoding of the 
+/// secp256k1 base point G as the X-coordinate. In that BIP the authors
+/// wrote the X-coordinate that is reproduced below.
+///
+/// [1]: https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#constructing-and-spending-taproot-outputs
+#[rustfmt::skip]
+pub const NUMS_X_COORDINATE: [u8; 32] = [
+    0x50, 0x92, 0x9b, 0x74, 0xc1, 0xa0, 0x49, 0x54,
+    0xb7, 0x8b, 0x4b, 0x60, 0x35, 0xe9, 0x7a, 0x5e,
+    0x07, 0x8a, 0x5a, 0x0f, 0x28, 0xec, 0x96, 0xd5,
+    0x47, 0xbf, 0xee, 0x9a, 0xce, 0x80, 0x3a, 0xc0,
+];
+
+/// The number of bitcoin blocks that must elapse from when the deposit
+/// transaction is confirmed before it can be reclaimed by the depositor.
+pub const LOCK_TIME: u32 = 150;
```

Comment:
How are we verifying that in the script? My rust analyzer doesn't seem to think it's being used.

Reply:
Oh yeah it's not being used. It's for the reclaim script stuff, which I'll start working on soon.

Reply:
Okay, roger, this addresses my comment on https://github.com/stacks-network/sbtc/pull/334 as well.